modules/stochastic_tools/src/samplers/ActiveLearningMonteCarloSampler.C

```diff
@@ -55,12 +61,23 @@ ActiveLearningMonteCarloSampler::sampleSetUp(const Sampler::SampleMode /*mode*/)
   if (_check_step == _step)
     return;
 
+  if (_is_sampling_completed)
+    mooseError("Internal bug: the adaptive sampling is supposed to be completed but another sample "
```

Comment:
Please add a test for this for coverage purposes.

Reply:
What kind of test are you thinking about? Usually, this error is hit when the implementation is incorrect (e.g., when you added a sampler but didn't include it in the list of `adaptiveSamplerNames`). In other words, this is a check to help future developers more than users.